bin/load-mocks

```diff
@@ -164,6 +164,8 @@ def create_system_time_series():
 def create_sample_time_series(event, release=None):
     if event is None:
         return
+    
+    Event.objects.bind_nodes([event], 'data')
```

Comment:
i don't think event is imported

Reply:
* `Event`

Reply:
Damn.

Reply:
Thanks, fixed in ef477f79f1fdbdbbc0c56990f0fceefb97b20435.